spec/ttml1.xml

```diff
@@ -7454,27 +7454,25 @@ associated (owning) element node.</p>
 <gitem>
 <label>[associate region]</label>
 <def>
-<p>A <loc href="#element-vocab-type-content">Content</loc> element is associated with a region according
-to the following ordered rules, where the first rule satisfied
-is used and remaining rules are skipped:</p>
+<p>A <loc href="#element-vocab-type-content">Content</loc> element <emph>E</emph> is associated 
+with a region <emph>R</emph> if
+any of the following assertions is true:</p>
```

Comment:
Why are you change from first satisfied rule to any satisfied rule? The existing text orders the rules, which may lead to a different result when the rules are not ordered. This seems problematic and unjustified.

Reply:
Clearly this (new) algorithm is broken since it permits an element to be associated with four different regions, one for each rule. The original issue [1] pointed out that an ambiguity existed, which your rewrite has not fixed, but instead made worse (by increasing the ambiguity).

[1] https://github.com/w3c/ttml1/issues/194#issue-115107118

Reply:
@skynavga see https://github.com/w3c/ttml1/issues/194#issuecomment-272434373 in which the group agreed that content elements can be associated with multiple regions simultaneously.

If you don't believe this is true, look again at https://github.com/w3c/ttml1/issues/194#issuecomment-268853318 and consider that a `<div>` element, for example, may set styles that are important for all of its temporally active descendants, and that those descendants may each be selected into a different region. Therefore that `div` itself must be selected into multiple regions.